src/class/object_genome_variants.php

```diff
@@ -351,9 +355,15 @@ function getForm ()
             }
         }
 
-        // Add genome build name to VOG/DNA field.
-        $this->aColumns['VariantOnGenome/DNA']['description_form'] = '<B>Relative to ' . $_CONF['refseq_build'] . ' / ' . $_SETT['human_builds'][$_CONF['refseq_build']]['ncbi_name'] . '.</B>' .
-            (!$this->aColumns['VariantOnGenome/DNA']['description_form']? '' : '<BR>' . $this->aColumns['VariantOnGenome/DNA']['description_form']);
+        // Add genome build name and element data to VOG/DNA fields, and add in the link to the HGVS website.
+        $aActiveBuilds = $_DB->query('SELECT id, name, column_suffix FROM ' . TABLE_GENOME_BUILDS)->fetchAllGroupAssoc();
+        foreach ($aActiveBuilds as $sBuild => $aBuild) {
+            $sColumn = 'VariantOnGenome/DNA' . (!$aBuild['column_suffix']? '' : '/' . $aBuild['column_suffix']);
+            $this->aColumns[$sColumn]['description_form'] = '<B>Relative to ' . $aBuild['name'] . '.</B>' .
+                (!$this->aColumns[$sColumn]['description_form']? '' : '<BR>' . $this->aColumns[$sColumn]['description_form']);
+            $this->aColumns[$sColumn]['element_data'] = array('genome_build' => $sBuild);
+            $this->aColumns[$sColumn]['form_type'][0] = str_replace('(HGVS format)', '(<A href="http://varnomen.hgvs.org/recommendations/DNA" target="_blank">HGVS format</A>)', $this->aColumns[$sColumn]['form_type'][0]);
+        }
```

Comment:
Add the 'Relative to <genome_build>' message in the Variant Creation Form, plus a link to varnomen.HGVS.org .